packages/picasso/src/utils/Modal/modal-manager.ts

```diff
@@ -17,11 +19,30 @@ export class ModalManager {
     }
 
     this.modals.splice(modalIndex, 1)
+    this.liftScrollLock()
   }
 
   isTopModal(modalId: number) {
     return (
       this.modals.length > 0 && this.modals[this.modals.length - 1] === modalId
     )
   }
+
+  private dropScrollLock() {
+    if (!isDefined(this.prevBodyOverflow)) {
```

Comment:
```suggestion
    if (!isDefined(this.prevBodyOverflow) && isBrowser()) {
```

I think we should prepare this SSR friendly and add `import { isBrowser } from '@toptal/picasso-shared'`

Reply:
I think its used inside useEffect, so we probably don't need the extra test? 🤔

Reply:
- [ ] Ensure scroll lock is only managed in browser environment